src/uu/seq/src/seq.rs

```diff
@@ -208,8 +111,8 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
     } else {
         1.0
     };
-    let step = if free.len() > 2 {
-        let slice = &free[1][..];
+    let step = if numbers.len() > 2 {
```

Comment:
I'd change `step` to `increment` to echo the usage text, but that's a minor nit and `step` is fine if you hold a differing/stronger opinion.

```suggestion
    let increment = if numbers.len() > 2 {
```